actions/setup-nextstrain-cli/action.yaml

```
@@ -0,0 +1,52 @@
+name: Setup Nextstrain CLI
+description: >-
+  This GitHub Actions action is intended to be called by workflows in our other
+  repos when they need a working copy of the Nextstrain CLI (the `nextstrain`
+  program) installed and don't particularly care about the exact Python
+  environment in which its installed.
+  
+  For example, this is typically the case when the workflow's primary step
+  invokes `nextstrain build` to run a build in a container runtime (locally via
+  Docker or remotely via AWS Batch).  Workflows that are more complex (e.g.
+  have additional dependencies to install, care about Conda, etc.) should
+  probably continue to install the Nextstrain CLI themselves for now.
+
+inputs:
+  cli-version:
+    description: >-
+      Version of Nextstrain CLI to install, as a dependency version spec
+      understood by Pip (i.e. conforming to PEP-0508).  Defaults to none, which
+      means the latest version available will be installed.
```

Not sure when this input would be used, but I could see myself naively specifying cli-version='3.2.4'` which would error.

```suggestion
  cli-version:
    description: >-
      Version of Nextstrain CLI to install, as a dependency version spec
      understood by Pip (i.e. conforming to PEP-0508) (e.g. >3.2 or ==3.2.4).  Defaults to none, which
      means the latest version available will be installed.
```

---

Added and repushed.